code/__DEFINES/economy.dm

```diff
@@ -0,0 +1,11 @@
+#define SPECIAL_CARGO	1
```

Comment:
Why are they not in a special associative list for speed rather than looping through everything?